### PR TITLE
Still more discoveries from the .cn IPRD list

### PR DESCRIPTION
**Pull request by AntiPontifex**
See title

### CODE REVIEW

**Comment by Dum4G** [comment]
tvg-id should not contain logos

File: streams/cn.m3u [hunk 1]
```diff
@@ -1447,3 +1447,43 @@ http://yslk.chinashadt.com:1635/live/stream:di1.stream/playlist.m3u8
 http://yslk.chinashadt.com:1635/live/stream:di2.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="",龙岩综合 (540p)
 http://stream.lytv.net.cn/2/sd/live.m3u8
+#EXTINF:-1 tvg-id="",Ankang TV-1 News [Geo-blocked]
+http://124.47.34.186/PLTV/88888888/224/3221225855/index.m3u8
+#EXTINF:-1 tvg-id="",Beijing Traffic Radio TV [Geo-blocked]
+http://123.56.24.28:1935/live/fm1039/96K/tzwj_video.m3u8
```
Comment (BellezaEmporium) [inline streams/cn.m3u:1453]: IPRD : http://lhttp.qingting.fm/live/336/64k.mp3

Comment (BellezaEmporium) [reply streams/cn.m3u:1453]: https://jwplay.hebyun.com.cn/live/hbystv/1500k/tzwj_video.m3u8 could be another live to add.

the correct FM could be https://brtv-radiolive.rbc.cn/alive/fm1039.m3u8

Comment (BellezaEmporium) [reply streams/cn.m3u:1453]: The original website is down (from my side) so I don't think that works anymore.

File: streams/cn.m3u [hunk 2]
```diff
@@ -1447,3 +1447,43 @@ http://yslk.chinashadt.com:1635/live/stream:di1.stream/playlist.m3u8
...
+#EXTINF:-1 tvg-id="",Binhai Life TV [Geo-blocked]
+http://60.30.52.41/live/bhtv2/playlist.m3u8
+#EXTINF:-1 tvg-id="",Binhai News TV [Geo-blocked]
+http://60.30.52.41/live/bhtv1/playlist.m3u8
+#EXTINF:-1 tvg-id="",Fengtai Culture & Life TV
+http://60.175.115.119:1935/live/wenhua/playlist.m3u8
+#EXTINF:-1 tvg-id="",Fengtai News TV
+http://60.175.115.119:1935/live/zonghe/playlist.m3u8
+#EXTINF:-1 tvg-id="",Foochow Children TV
+http://live.zohi.tv/video/s10001-sepd-4/index.m3u8
```
Comment (BellezaEmporium) [inline streams/cn.m3u:1469]: As for Zohi : http://live.zohi.tv/video/s10001-fztv-1/index.m3u8
http://live.zohi.tv/video/s10001-fztv-3/index.m3u8
http://live.zohi.tv/video/s10001-fztv-4/index.m3u8

Radio : http://live.zohi.tv/audio/s10001-FM944/index.m3u8 Fuzhou (?) News Radio
http://live.zohi.tv/audio/s10001-FM893/index.m3u8 Fuzhou Music Radio
http://live.zohi.tv/audio/s10001-FM876/index.m3u8 Fuzhou Traffic Radio
http://live.zohi.tv/audio/s10001-FM901/index.m3u8 Fuzhou ? Radio

File: streams/cn.m3u (hunk 1 above)
Comment (AntiPontifex) [reply streams/cn.m3u:1453]: One of the (many) problems I'm having with this list is geo-blocking. Secondly, occassionally "odd" things are happening when I attempt to access some of these sites (e.g. weird pop-ups, ads), so what I'm going to be doing to keep this going is to tread lightly and accept reviewed recommendations automatically.

**Pull request closed without merging**